libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -620,7 +620,7 @@ class AP_AHRS {
 
     // return primary accels, for lua
```

Comment:
We should probably remote the "for lua". This is used in loads of places now.

Reply:
https://github.com/ArduPilot/ardupilot/blob/da69e2267392788b709eefa24395595a98301921/ArduPlane/pullup.cpp#L149

https://github.com/ArduPilot/ardupilot/blob/da69e2267392788b709eefa24395595a98301921/ArduPlane/servos.cpp#L643

https://github.com/ArduPilot/ardupilot/blob/da69e2267392788b709eefa24395595a98301921/libraries/AP_AHRS/AP_AHRS_Logging.cpp#L107

https://github.com/ArduPilot/ardupilot/blob/da69e2267392788b709eefa24395595a98301921/libraries/AP_IBus_Telem/AP_IBus_Telem.cpp#L599

https://github.com/ArduPilot/ardupilot/blob/da69e2267392788b709eefa24395595a98301921/libraries/AP_IBus_Telem/AP_IBus_Telem.cpp#L603

https://github.com/ArduPilot/ardupilot/blob/da69e2267392788b709eefa24395595a98301921/libraries/AP_IBus_Telem/AP_IBus_Telem.cpp#L607

Reply:
Fixed thanks